src/System.Drawing.Design.Primitives/ref/System.Drawing.Design.Primitives.cs

```diff
@@ -7,6 +7,7 @@
 
 namespace System.Drawing.Design
 {
```

Comment:
Using statements outside namespace. Please take a look at the style guide in this repo and follow that.

Reply:
Actually, you shouldn't be manually editing this file most of the time.  Try msbuild /t:GenerateReferenceSource on the src project.